### PR TITLE
WIP: Minor fixes and update to new ink! version

### PR DESCRIPTION
**Pull request by athei**
I am currently working through the contracts workshop as part of my onboarding. I am just fixing some minor oversights that I notice while doing that.

### CODE REVIEW

File: 1/storing-a-mapping.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ mod mycontract {
         // Get the value for the calling AccountId
         #[ink(message)]
         fn get_my_number(&self) -> u32 {
-            let caller = env.caller();
+            let caller = self.env.caller();
```
Comment (Robbepop) [inline 1/storing-a-mapping.md]:
```suggestion
            let caller = self.env().caller();
```

File: 1/incrementing-the-value.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ impl MyContract {
 }
 ```
 
-However, if the value is not initialized, your contract will compile fine, but will panic during contract execution! **We really cannot understate how easy it is to make mistakes this way.**
+However, if the value is not initialized, your contract will compile fine, but will panic during contract execution! **We really cannot overstate how easy it is to make mistakes this way.**
```
Comment (athei) [inline 1/incrementing-the-value.md]: Now that I think about it, it seems to be actually correct. I think I got it backwards.

File: 1/storing-a-mapping.md (hunk 1 above)
Comment (athei) [reply 1/storing-a-mapping.md]: Yep good spot.

**Comment by athei** [comment]
> Thank you for the fixes. However, I guess there are many more things that need to be fixed for our ink! workshop.

One step at a time I guess. Right now, I only intend on fixing things that somehow blocked or confused me while doing the workshop.

**Comment by Robbepop** [comment]
We can merge as soon as you press "Ready for review". xD

**Comment by athei** [comment]
I guess I will put the coming changes into another PRs then. I just removed the commit regarding the `overstate` vs `understate` and squashed the fixup. Yea just merge this micro pr when you are ready.

**Comment by Robbepop** [comment]
Thank you for your help in keeping this updated!

**Pull request merged**